### DISCUSSION TITLE
arrivalRate > 3 causes node TimeoutOverflowWarning

### DISCUSSION THREAD

**Discussion by tchambers97** [artilleryio/artillery#1744] [Artillery]
Hi, I'm experimenting with load phases to get to 50 concurrent users for the `Sustained` phase. An `arrivalRate: 2` didn't quite do it, so I bumped it up to `arrivalRate: 5`, which threw this node error:
`(node:12803) TimeoutOverflowWarning: Infinity does not fit into a 32-bit signed integer.`
`Timeout duration was set to 1.` 

The maximum I can set it to is 3. Any idea why that might be? Here is the full phase config:
  phases:
    - duration: 1200
      rampTo: 3
      maxVusers: 14
      name: Ramp Up
    - duration: 5400
      arrivalRate: 3
      maxVusers: 50
      name: Sustained

The test failed right away and did not execute the `Ramp Up` phase either.

**Comment by hassy** [comment]
hi @tchambers97 👋 thank you for the bug report - it's been fixed in https://github.com/artilleryio/artillery/pull/1683 - we're releasing a new version of Artillery tomorrow which will include the fix.
